gmso/utils/compatibility.py

```diff
@@ -46,6 +50,7 @@ def check_compatibility(topology, accepted_potentials):
 def _check_single_potential(potential, accepted_potentials):
     """Check to see if a single given potential is in the list of accepted potentials."""
     for ref in accepted_potentials:
+        print(ref.independent_variables, potential.independent_variables)
```

Comment:
Forgotten print statement

Reply:
Fixed in commit 93d813f82a66dc81a825042b283a26b3b4890c96.